workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/CodeWriter/Scope.cs

```diff
@@ -103,7 +113,12 @@ public static TypeBlock Type(string declaration, Action<TypeBlock> populate)
 
         public static TypeBlock AnnotatedType(string annotation, string declaration, Action<TypeBlock> populate)
         {
-            return new TypeBlock(declaration, populate, annotation);
+            return new TypeBlock(declaration, populate, new List<string> { annotation });
+        }
+
+        public static TypeBlock AnnotatedType(List<string> annotations, string declaration, Action<TypeBlock> populate)
```

Comment:
Same with other func signatures, use `IEnumerable<string>` instead of concrete List